examples/tutorial3/tutorial/app.py

```diff
@@ -9,18 +9,20 @@ def startup(self):
         self.webview = toga.WebView(style=Pack(flex=1))
         self.url_input = toga.TextInput(
             initial='https://github.com/',
-            style=Pack(flex=1, padding=5)
+            style=Pack(flex=1)
```

Comment:
At column 19: (F405) 'Pack' may be undefined, or defined from star imports: toga.style.pack